src/main/java/hudson/matrix/MatrixConfiguration.java

```diff
@@ -553,4 +553,36 @@ public MatrixBuild getMatrixBuild() {
     public void doConfigure(StaplerResponse rsp) throws IOException {
         rsp.sendError(HttpServletResponse.SC_NOT_FOUND);
     }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Why would you need it for bumping the Parent POM declaration?

Reply:
Sorry, It's been a mistake. MatrixProject and MatrixConfiguration shouldn't been committed neither pulled.